Added Common Troubleshooting Tip to README

shford opened the pull request:
Pretty much what the title says. I experienced this issue with the on-screen keyboard not working for entering the Host PC IP address and expect that it's fairly common. 

I found several old posts in forums with different advice so I decided to type up what worked.

I have no affiliation with Samsung or the SmartThings app; it just seemed like the easiest way to get a virtual numeric keyboard paired with my Samsung TV. I didn't test input with a Bluetooth keyboard as I don't own one, but there's reports of that working too.

Thanks for making this repository - it worked like a charm! I hope my small troubleshooting contribution can help even just a few more folks get their setup working. Cheers

Georift commented on README.md:
````diff
@@ -29,12 +32,13 @@ For documentation about the Jellyfin app, see [here](https://github.com/jellyfin
 
 ```bash
-docker run --rm ghcr.io/georift/install-jellyfin-tizen <samsung tv ip> <build option> <tag url>
+**docker run --rm ghcr.io/georift/install-jellyfin-tizen** <u>samsung tv ip</u> [build option] [tag url]
````
Could you revert the `<u>` and `**` changes please? I don't think they render correctly in github. But I might be mistaken.

I like the `[]` denoting option though! 🙂

shford commented:
Removed broken github markup syntax. 

I added to/reformatted/cleaned up a lot of the README. It think it looks pretty slick now. Lmk what you think.

...also sorry for all the commits. I had to push and reload github to make sure things were actually rendering as expected.

Georift reviewed (approved):
Looks great, thanks for all these changes.

Might be worth pulling the on-screen keyboard troubleshooting off into another page, it's starting to get really long.

But this is awesome so I'll merge it now. Thanks again.
